index.js

```diff
@@ -1,253 +1,38 @@
 
-var fs = require('fs'),
-    levelup = require('levelup'),
-    multilevel = require('level-multiply'),
-    parser = require('osm-pbf-parser')(),
-    through = require('through2'),
-    suggester = require('pelias-suggester-pipeline'),
-    propStream = require('prop-stream'),
-    settings = require('pelias-config').generate(),
-    dbclient = require('pelias-dbclient')(),
-    OsmiumStream = require('osmium-stream');
-
-var Backend = require('geopipes-elasticsearch-backend');
-
-// esclient.livestats();
+/**
+  The openstreetmap importer provides a way of parsing, mapping and
+  augmenting OSM data in to elasticsearch.
+**/
 
-var backend = function( index, type ){
-  return new Backend( dbclient.client, index, type );
+var elasticsearch = require('pelias-dbclient'),
+    adminLookup = require('pelias-admin-lookup'),
+    suggester = require('pelias-suggester-pipeline'),
+    dbmapper = require('./stream/dbmapper');
```

Comment:
why are all the stream object defined in this project required and saved as part of the osm object except dbmapper? would we expect the client of this module to want access to all the other objects but not dbmapper, or adminLookup/suggester for that matter? just curious what the distinction is here.